server/server.go

```diff
@@ -0,0 +1,251 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+	"strings"
+	"time"
+
+	grpc_logrus "github.com/grpc-ecosystem/go-grpc-middleware/logging/logrus"
+	ilogger "github.com/meateam/elasticsearch-logger"
+	pb "github.com/meateam/fav-service/proto"
+	"github.com/meateam/fav-service/service"
+	"github.com/meateam/fav-service/service/mongodb"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"go.elastic.co/apm/module/apmmongo"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+	"go.mongodb.org/mongo-driver/x/mongo/driver/connstring"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/health"
+	"google.golang.org/grpc/health/grpc_health_v1"
+)
+
+const (
+	configPort                  		= "port"
+	envPrefix 							= "FVS"
+	configHealthCheckInterval          	= "health_check_interval"
+	configMongoConnectionString 		= "mongo_host"
+	configMongoClientConnectionTimeout 	= "mongo_client_connection_timeout"
+	configMongoClientPingTimeout       	= "mongo_client_ping_timeout"
+	configElasticAPMIgnoreURLS         	= "elastic_apm_ignore_urls"
+
+
+)
+
+func init() {
+	viper.SetDefault(configPort, "8080")
+	viper.SetDefault(configHealthCheckInterval, 3)
+	viper.SetDefault(configElasticAPMIgnoreURLS, "/grpc.health.v1.Health/Check")
+	viper.SetDefault(configMongoConnectionString, "mongodb://mongo:27017/favorite")
```

Comment:
```suggestion
	viper.SetDefault(configMongoConnectionString, "mongodb://localhost:27017/favorite")
```

Reply:
But if I run with docker it should be mongo

Reply:
Correct. However, the default values are for the local execution. Therefor, it suppose to be `localhost` instead of `mongo` and when you dockerize it you should use env var to insert the `mongodb://mongo:27017/favorite`

Reply:
Ok